DirectProgramming/C++/CompilerInfrastructure/OpenMP_Offload_Features/License.txt

```diff
@@ -1,8 +1,19 @@
-Copyright Intel Corporation
+Copyright (c) Intel Corporation
```

Comment:
Why the update? The (c) is not needed per legal

Reply:
Legal has been inconsistent. Some are saying no (c) required. Some are saying (c) without a year. And some are saying (c) with first year, but no subsequent years required. So I've been going with the middle ground.